apps/docs/pages/guides/functions/troubleshooting.mdx

```diff
@@ -34,6 +34,31 @@ There are two debugging tools available: Invocations and Logs. Invocations shows
 - Search the [Edge Runtime](https://github.com/supabase/edge-runtime) and [CLI](https://github.com/supabase/cli) repos for the error message, to see if it has been reported before.
 - If the output from the commands above does not help you to resolve the issue, please open a support ticket via the Supabase Dashboard (by clicking the "Help" button at the top right) and include all output and details about your commands.
 
+# Advanced Techniques
+
+## Finding Bundle Size
+
+To find the bundle size of a function, run the following command locally:
+
+```bash
+deno info /path/to/function/index.ts
```

Comment:
This doesn't give the accurate eszip bundle size. The size we show when they deploy the function is the accurate value. We can mention both - "you can find the accurate bundle size when you deploy your function via Supabae CLI".